raft/src/test/java/org/apache/kafka/raft/RaftUtilTest.java

```diff
@@ -646,6 +647,30 @@ public void testAddRaftVoterRequestAckWhenCommittedNotIgnorable() {
         );
     }
 
+    @Test
+    public void testAddVoterResponse() {
+        for (Errors error : Errors.values()) {
```

Comment:
Iterating over all errors should be unnecessary, but it's fine since it runs very quickly